Client/src/Context/UserContext.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+import { getUserFromStorage, putUserInStorage } from '../Common/Storage'
+
+import { IUserAccountModel } from '../Models/Authentication/IUserAccountModel'
+
+interface IUserContextProps {
+  user: IUserAccountModel
+  setUserContext: (user: IUserAccountModel) => void
+}
+
+interface IProps {
+  children: any
+}
+
+export const UserContext = React.createContext<Partial<IUserContextProps>>({})
+
+export const UserContextWrapper = (props: IProps) => {
```

Comment:
Could you add this to the root? Similar call as variables & dimensions and lower overhead. We wrap app.ts with 2 different context at the moment